src/routes/(app)/admin[[hash=admin_hash]]/seo/+page.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+	export let data;
+	export let form;
+</script>
+
+<h1 class="text-2xl">SEO Config</h1>
+{#if form?.success}
+	<div class="alert alert-success">{form.success}</div>
+{/if}
+<form method="post" class="flex flex-col gap-4" action="?/update">
+	<label class="checkbox-label">
+		<input type="checkbox" name="enableSEO" class="form-checkbox" checked={data.enableSEO} />
+		Enable SEO cloak
```

Comment:
Please work on the naming of the vars

"Enable SEO cloak" = disable SEO

Rename the variable to `hideFromSearchEngines`